### PR TITLE
Moved GProcessID, GProcessName and GCaptureTimePoint to CaptureData

### PR DESCRIPTION
**Pull request by florian-kuebler**

### CODE REVIEW

File: OrbitCore/CaptureData.h [hunk 1]
```diff
@@ -29,10 +32,24 @@ class CaptureData {
     return selected_functions_;
   }
 
+  [[nodiscard]] int32_t process_id() { return process_id_; }
+
+  [[nodiscard]] const std::string& process_name() { return process_name_; }
+
+  [[nodiscard]] const std::chrono::system_clock::time_point&
+  capture_time_point() {
```
Comment (dimitry-) [inline OrbitCore/CaptureData.h:40]: let's take this opportunity to rename this to capture_start_time

File: OrbitCore/Capture.cpp [hunk 2]
```diff
@@ -81,16 +74,12 @@ void Capture::FinalizeCapture() {
 }
 
 void Capture::ClearCaptureData() {
-  GProcessId = -1;
-  GProcessName = "";
-  GThreadNames.clear();
```
Comment (dimitry-) [inline OrbitCore/Capture.cpp:86]: what happened to threadnames?

**Comment by dimitry-** [comment]
and I believe this fixes http://b/163297639

File: OrbitCore/Capture.cpp (hunk 2 above)
Comment (florian-kuebler) [reply OrbitCore/Capture.cpp:86]: Ah thanks! That was by accident.

File: OrbitCore/CaptureData.h (hunk 1 above)
Comment (florian-kuebler) [reply OrbitCore/CaptureData.h]: done.

**Comment by florian-kuebler** [comment]
> and I believe this fixes http://b/163297639

I'll have a look.

**Comment by florian-kuebler** [comment]
> > and I believe this fixes http://b/163297639
> 
> I'll have a look.

Checked it, and yes process name is shown correctly again.

**Pull request merged**